content/sensu-go/5.14/guides/clustering.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "How to run a Sensu cluster"
+linkTitle: "Running a Sensu Cluster"
+description: "Clustering is important to make Sensu more highly available, reliable, and durable. It can help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents. Read the guide to configure a Sensu cluster."
+weight: 120
+version: "5.13"
+product: "Sensu Go"
+platformContent: false
+menu: 
+  sensu-go-5.13:
+    parent: guides
+---
+
+- [What is a Sensu cluster?](#what-is-a-sensu-cluster)
+- [Why use clustering?](#why-use-clustering)
+- [Configuring a cluster](#configuring-a-cluster)
+- [Adding sensu agents to the cluster](#adding-sensu-agents-to-the-cluster)
+- [Cluster health](#cluster-health)
+- [Managing cluster members](#add-a-cluster-member)
+- [Security](#security)
+  - [Client-to-server transport security with HTTPS](#client-to-server-with-https)
+  - [Client-to-server authentication with HTTPS client certificates](#client-to-server-auth-with-https)
+  - [Peer communication authentication with HTTPS client certificates](#peer-auth-https)
+  - [Sensu agent with HTTPS](#sensu-agent-https)
+- [Using an external etcd cluster](#using-an-external-etcd-cluster)
+- [Troubleshooting](#troubleshooting)
+
+## What is a Sensu cluster?
+
+A Sensu cluster is a group of [at least three][1] sensu-backend nodes, each connected to a shared etcd cluster, using Sensu's embedded etcd or an external etcd cluster. Creating a Sensu cluster ultimately configures an [etcd cluster][2].
+
+## Why use clustering?
+
+Clustering is important to make Sensu more highly available, reliable, and durable. It will help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents.
```

Comment:
```suggestion
Clustering makes Sensu more highly available, reliable, and durable. It will help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents.
```